docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -603,6 +604,41 @@ We recommend that you:
   - Conda env will export or create environments based on a file with
     conda and pip requirements
 
+.. _set-env-vars:
+
+Setting environment variables
+=============================
+
+If you want to associate environment variables with an environment,
+you can use the config API. This is recommended as an alternative to
+using activate and deactivate scripts since those are an execution of
+arbitrary code that may not be safe.
+
+First, create your environment and activate it::
+
+  Conda create -n test-env
+  Conda activate test-env
+
+To list any variables you may have, run ``conda env config vars list``.
+
+To set environment variables, run ``conda env config vars set my_var=value``.
+
+Once you have set an environment variable, you have to reactivate your environment:
+``conda activate test-env``
+
+To check if the environment variable has been set, run
+``echo my_var`` or ``conda env config vars list``.
+
+When you deactivate your environment, you can use those same commands to see that
+the environment variable goes away.
+
+To unset the environment variable, run ``conda env config vars unset my_var -n test-env``.
```

Comment:
This might be a nice place to talk about specifying an environment using the `-n/--name` or `-p/--prefix` flags. Some thing like "You can specify the environment you want to effect using the -n and -p flags". Then the bit that uses that information to unset some vars "``conda env config vars unset my_var -n test-env``."

Reply:
I added the line in, but I wanted to make sure I put it in the right place and to check if it needs more info.

Reply:
Looks good. In the cmd line help bit it says

```
Target Environment Specification:
  -n ENVIRONMENT, --name ENVIRONMENT
                        Name of environment.
  -p PATH, --prefix PATH
                        Full path to environment location (i.e. prefix).
```
I think it would be nice to note that `-n` allows users to specify the name of their environment and `-p` allows them to specify the path to an environment. Once that is in, I think this is good to merge!